tests/network-setups/fabric/dev/scripts/setupCCLocal.sh

```diff
@@ -2,6 +2,7 @@ directory=$(dirname $0)
 
 CHAINCODE_PATH=$directory/../../../fabric/shared/chaincode
 INTEROP_CC_PATH=$PWD/../../../../core/network/fabric-interop-cc
+INTEROPCC_CHAINCODE_PATH=$PWD/../../../../core/network
```

Comment:
`INTEROP_CC_PATH` can be used I guess instead of defining new `INTEROPCC_CHAINCODE_PATH`

Reply:
Done @sanvenDev .